### PR TITLE
Update m5stack_dial.md with Adafruit purchase link

### PR DESCRIPTION
**Pull request by RetiredWizard**
The Adafruit project page should probably also be updated to indicate that there is CircuitPython support but I don't know how to make that update.

### CODE REVIEW

**Comment by makermelissa** [comment]
> The Adafruit project page should probably also be updated to indicate that there is CircuitPython support but I don't know how to make that update.

Add the product URL under board_url:

**Comment by RetiredWizard** [comment]
Sorry @makermelissa, I thought I was being clever and skipped re-reading the instructions. Hopefully this is what you need 😁

**Comment by makermelissa** [comment]
Can you please leave both URLs? It's a list, so it can have multiple URLs like this:
```
board_url:
  - "url1"
  - "url2"
 ```

File: _board/m5stack_dial.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: "Dial - Smart Rotary Knob with Touch Screen Download"
 name: "Dial - Smart Rotary Knob with Touch Screen"
 manufacturer: "M5Stack"
 board_url:
- - "https://docs.m5stack.com/en/core/M5Dial"
+ - "https://www.adafruit.com/product/5966"
```
Comment (makermelissa) [inline _board/m5stack_dial.md:8]:
```suggestion
 - "https://docs.m5stack.com/en/core/M5Dial"
 - "https://www.adafruit.com/product/5966"
```

**Review by makermelissa — CHANGES_REQUESTED** [review]
By having both, we can use the data in other ways in the future.

**Review by makermelissa — APPROVED** [review]
Thank you. Looks great.

**Pull request merged**